api/src/test/java/org/openmrs/module/registrationapp/converter/RegistrationSummaryExtensionsGeneratorTest.java

```diff
@@ -73,22 +78,23 @@ public void generate_shouldGenerateRegSummaryFromRegApp() throws Exception {
     
     @Test(expected = IllegalArgumentException.class)
     public void generate_shouldThrowWhenNotRegAppConfig() throws Exception {
-    	InputStream inputStream = getClass().getClassLoader().getResourceAsStream("registration_app.json");    	
-    	List<AppDescriptor> appDescriptors = new ObjectMapper().readValue(inputStream, new TypeReference<List<AppDescriptor>>() {});
+    	AppDescriptor appDescriptor = new AppDescriptor("fooId", "fooDesc", null, null, null, null, 0);
+    	appDescriptor.setInstanceOf("not.a.reg.app");
     	
-    	AppDescriptor appDescriptor = appDescriptors.get(0);
-    	appDescriptor.setId("acme.sysadmin.manage");
     	List<Extension> extensions = RegistrationSummaryExtensionsGenerator.generate(appDescriptor);
     }
     
     @Test
-    public void generate_shouldRetrunEmptyWhenEmptySectionsInConfig() throws Exception {
-    	InputStream inputStream = getClass().getClassLoader().getResourceAsStream("registration_app.json");    	
-    	List<AppDescriptor> appDescriptors = new ObjectMapper().readValue(inputStream, new TypeReference<List<AppDescriptor>>() {});
+    public void generate_shouldReturnEmptyWhenEmptySectionsInConfig() throws Exception {
+    	AppDescriptor appDesc = new AppDescriptor("my.registrationapp.registerPatient", "Create a new Patient Record", "Register Patient", null, null, null, 0, null, null);
+    	appDesc.setInstanceOf("registrationapp.registerPatient");
     	
-    	AppDescriptor appDescriptor = appDescriptors.get(0);
-    	appDescriptor.setConfig(appDescriptor.getConfig().putObject("section"));
-    	List<Extension> extensions = RegistrationSummaryExtensionsGenerator.generate(appDescriptor);
-    	assertEquals(true, extensions.isEmpty());
+    	ObjectNode config = new ObjectMapper().createObjectNode();
+    	config.putArray("sections"); 
+    	appDesc.setConfig(config);
+
+    	List<Extension> regSummaryExtensions = RegistrationSummaryExtensionsGenerator.generate(appDesc);
+
+    	assertThat(regSummaryExtensions, is(empty()));
```

Comment:
Same remark, try to adopt the **setup / replay / verif** breakdown comments.

Reply:
Sure @mks-d, thank you.